WordPress/src/jetpack/assets/support_chat_widget.js

```diff
@@ -0,0 +1,31 @@
+;(window.DocsBotAI = window.DocsBotAI || {}),
+    (DocsBotAI.init = function (c) {
+      return new Promise(function (e, o) {
+        var t = document.createElement('script')
+        ;(t.type = 'text/javascript'), (t.async = !0), (t.src = 'https://widget.docsbot.ai/chat.js')
+        var n = document.getElementsByTagName('script')[0]
+        n.parentNode.insertBefore(t, n),
+          t.addEventListener('load', function () {
+            window.DocsBotAI.mount({
+              id: c.id,
+              supportCallback: c.supportCallback,
+              identify: c.identify,
```

Comment:
As I understand, we don't yet pass any of these parameters besides `id`? We would be passing them from the native code, correct?

Reply:
We can pass a host of options.  Need to determine what is required for this.